acstools/acszpt.py

```diff
@@ -132,13 +133,16 @@ class Query:
                  F140LP, F150LP, F165LP
 
     """
+
     def __init__(self, date, detector, filt=None):
 
         # Set the attributes
         self._date = date
         self._detector = detector.upper()
         self._filt = filt
```

Comment:
Change this to `self._filt = filt.upper()` so that lowercase filters don't trip things up, and to be consistent with the treatment of `detector`.

Reply:
inserted a bug trying to fix this, working on cleaning it up

Reply:
(swapping to -> self._filt = filt.upper() breaks if filt = None)